src/pkg/path/resource_path.go

```diff
@@ -93,13 +96,16 @@ var serviceCategories = map[ServiceType]map[CategoryType]struct{}{
 		EmailCategory:    {},
 		ContactsCategory: {},
 		EventsCategory:   {},
+		DetailsCategory:  {},
```

Comment:
I think it would be best to make a separate validate function for metadata paths (or a separate function to get details paths, whatever makes more sense). Thus far I've been trying to make it difficult to construct a path that is incorrect. Adding details to this map will allow other packages to construct paths like `tenant-id/service/user-id/details` and add regular item data there